app/controllers/catalog_controller.rb

```diff
@@ -199,12 +199,12 @@ def set_solr_search_fields
       config.add_search_field('date_created') do |field|
         field.label = "Date"
         field.solr_parameters = {
-          :"spellcheck.dictionary" => "date_created"
+          :"spellcheck.dictionary" => "date_accepted"
         }
-        solr_name = Solrizer.solr_name("created", :stored_searchable)
+        field_included = Solrizer.solr_name("created", :stored_searchable)
         field.solr_local_parameters = {
-          qf: solr_name,
-          pf: solr_name
+          qf: "#{field_included} date_accepted_tesim",
```

Comment:
Maybe cleaner to do this:
https://github.com/ualbertalib/HydraNorth/blob/Sprint43/app/controllers/catalog_controller.rb#L169
and then use just field_included here? @criedlberger